app/config.py

```diff
@@ -220,7 +221,7 @@ class Config(object):
         # app/celery/scheduled_tasks.py
         "run-scheduled-jobs": {
             "task": "run-scheduled-jobs",
-            "schedule": crontab(minute=1),
+            "schedule": crontab(),
```

Comment:
Run every minute, not once per hour at the first minute. Jobs can now be scheduled down to the minute if people feel like it.

Compatible with the scheduled task https://github.com/cds-snc/notification-api/blob/14fc8db0c58dc22b3fb13cf91ad383b1897930a3/app/dao/jobs_dao.py#L72-L94

Reply:
Thank you Antoine! That fixes the issue I was having.